eZ/Bundle/EzPublishCoreBundle/EventListener/ConsoleCommandListener.php

```diff
@@ -42,9 +48,10 @@ class ConsoleCommandListener implements EventSubscriberInterface, SiteAccessAwar
     /**
      * ConsoleCommandListener constructor.
      */
-    public function __construct($defaultSiteAccessName, EventDispatcherInterface $eventDispatcher)
+    public function __construct($defaultSiteAccess, $siteAccessList, EventDispatcherInterface $eventDispatcher)
```

Comment:
@andrerom is correct.  I guess `%ezpublish.siteaccess.default%` is just a name so I will revert back.